Gruntfile.js

```diff
@@ -215,11 +215,33 @@ module.exports = function( grunt ) {
 			},
 			contributors: {
 				command: [
-					'echo "Generating contributor list since <%= fromDate %>"',
+					'echo "<h2>WooCommerce Admin</h2>" > contributors.html',
+					'echo "Generating contributor list for WC Admin since <%= fromDate %>"',
+					'./node_modules/.bin/githubcontrib --owner woocommerce --repo woocommerce-admin --fromDate <%= fromDate %>' +
+					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format html --sortOrder desc' +
+					' --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "<h2>WooCommerce Blocks</h2>" >> contributors.html',
+					'echo "Generating contributor list for WC Blocks since <%= fromDate %>"',
+					'./node_modules/.bin/githubcontrib --owner woocommerce --repo woocommerce-gutenberg-products-block' +
+					' --fromDate <%= fromDate %> --authToken <%= authToken %> --cols 6 --sortBy contributions --format html' +
+					' --sortOrder desc --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "<h2>Action Scheduler</h2>" >> contributors.html',
+					'echo "Generating contributor list for Action Scheduler since <%= fromDate %>"',
+					'./node_modules/.bin/githubcontrib --owner woocommerce --repo action-scheduler --fromDate <%= fromDate %>' +
+					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format html --sortOrder desc' +
+					' --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "<h2>REST API</h2>" >> contributors.html',
+					'echo "Generating contributor list for REST API since <%= fromDate %>"',
+					'./node_modules/.bin/githubcontrib --owner woocommerce --repo woocommerce-rest-api --fromDate <%= fromDate %>' +
+					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format html --sortOrder desc' +
+					' --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "<h2>WooCommerce core</h2>" >> contributors.html',
```

Comment:
I think we should move core to the first, like we does with the changelogs.

Reply:
I wanted to politely give credit to our guests first, but we can swap, sure.